api/views/notifications/notifications.py

```diff
@@ -0,0 +1,23 @@
+from flask import Blueprint, make_response, jsonify
+
+from api.my_flask import app
+
+
+class NotificationsBlueprint(Blueprint):
+    pass
+
+
+bp = NotificationsBlueprint('notifications', __name__)
+
+
+@bp.before_request
+def set_config():
+    global app_config, user_notifications
+    app_config = app.config
+    user_notifications = app_config.redis_user_notifications
+
+
+@bp.route('/unsubscribe_from_draft_errors_emails/<path:draft_name>/<path:email>', methods=['GET'])
```

Comment:
just `unsubscribe` should be enough instead of `unsubscribe_from_draft_errors_emails`